### PR TITLE
Fix wire currency sum on transfers tab

### PR DESCRIPTION
**Pull request by Luke-Oldenburg**
## Summary of the problem
The sum on the transfers page does not account for wire transfers in currencies other than USD.
See https://hackclub.slack.com/archives/C068U0JMV19/p1732644327652379

## Describe your changes
I used the Money.from_cents function to convert the values to USD.

### CODE REVIEW

**Review by sampoder — CHANGES_REQUESTED** [review]
Instead of doing conversions, let's use `Wire#usd_amount_cents` and sum using that, after https://github.com/hackclub/hcb/pull/8745 merges

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@ def index
   def show
     authorize @event
 
-    if !Flipper.enabled?(:event_home_page_redesign_2024_09_21, @event) && !(params[:event_home_page_redesign_2024_09_21] && admin_signed_in?)
+    if !Flipper.enabled?(:event_home_page_redesign_2024_09_21, @event) && !(params[:event_home_page_redesign_2024_09_21] && admin_signed_in?) || @event.demo_mode?
```
Comment (sampoder) [inline app/controllers/events_controller.rb]:
```suggestion
    if !Flipper.enabled?(:event_home_page_redesign_2024_09_21, @event) && !(params[:event_home_page_redesign_2024_09_21] && admin_signed_in?)
```

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -72,7 +72,7 @@ def show
     @money_out = all_transactions.reject { |t| t.amount_cents >= 0 }.first(3)
 
     @activities = PublicActivity::Activity.for_event(@event).order(created_at: :desc).first(5)
-    @organizers = @event.organizer_positions.includes(:user).order(users: { preferred_name: :asc, full_name: :asc })
+    @organizers = @event.organizer_positions.includes(:user).order(created_at: :desc)
```
Comment (sampoder) [inline app/controllers/events_controller.rb]:
```suggestion
    @organizers = @event.organizer_positions.includes(:user).order(users: { preferred_name: :asc, full_name: :asc })
```

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -90,19 +90,13 @@ def transaction_heatmap
     end
   end
 
-  def top_merchants
+  def merchants_categories
```
Comment (sampoder) [inline app/controllers/events_controller.rb]:
```suggestion
  def top_merchants
```

File: app/controllers/events_controller.rb [hunk 4]
```diff
@@ -90,19 +90,13 @@ def transaction_heatmap
...
-    end
-  end
-
-  def top_categories
-    authorize @event
     @categories = BreakdownEngine::Categories.new(@event).run
+
     respond_to do |format|
-      format.html { render partial: "events/home/top_categories", locals: { categories: @categories, event: @event } }
+      format.html { render partial: "events/home/merchants_categories", locals: { merchants: @merchants, categories: @categories, event: @event } }
```
Comment (sampoder) [inline app/controllers/events_controller.rb]:
```suggestion
      format.html { render partial: "events/home/top_categories", locals: { categories: @categories, event: @event } }
```

File: app/controllers/events_controller.rb [hunk 5]
```diff
@@ -149,7 +143,7 @@ def transactions
     @maximum_amount = params[:maximum_amount].presence ? Money.from_amount(params[:maximum_amount].to_f) : nil
     @missing_receipts = params[:missing_receipts].present?
 
-    @organizers = @event.organizer_positions.includes(:user).order(users: { preferred_name: :asc, full_name: :asc })
+    @organizers = @event.organizer_positions.includes(:user).order(created_at: :desc)
```
Comment (sampoder) [inline app/controllers/events_controller.rb]:
```suggestion
    @organizers = @event.organizer_positions.includes(:user).order(users: { preferred_name: :asc, full_name: :asc })
```

**Review by sampoder — CHANGES_REQUESTED** [review]
Just a couple of unrelated changes to revert, LGTM!

**Review by sampoder — APPROVED** [review]
LGTM!

**Comment by Luke-Oldenburg** [comment]
I don't even know where those changes came from lol

**Comment by sampoder** [comment]
¯\\_(ツ)_/¯

**Pull request merged**